contrib/debian/netdata-plugin-perf.postinst

```diff
@@ -9,14 +9,10 @@ case "$1" in
 
     if capsh --supports=cap_perfmon 2>/dev/null; then
         setcap cap_perfmon+ep /usr/libexec/netdata/plugins.d/perf.plugin
```

Comment:
should we `if !` this setcap too?

Reply:
We probably should actually.

If the system itself supports the capability, but the filesystem does not for some reason, this will still fail. Such a situation is _extremely_ unlikely, but still technically possible.